typescript/ws-protocol/README.md

```diff
@@ -1,8 +1,6 @@
-# Foxglove Studio WebSocket protocol
+# Foxglove WebSocket server and client
 
-This package provides server and client implementations of the [Foxglove Studio WebSocket protocol](https://github.com/foxglove/ws-protocol), enabling [Foxglove Studio](https://github.com/foxglove/studio) to ingest arbitrary “live” streamed data.
-
-The protocol is encoding-agnostic, i.e. it can support Protobuf messages, JSON messages, etc. (as long as the desired encoding is supported by both client and server).
+This package provides an example server implementation of the [Foxglove WebSocket protocol](https://github.com/foxglove/ws-protocol), enabling [Foxglove Studio](https://github.com/foxglove/studio) to ingest arbitrary “live” streamed data.
```

Comment:
Same as the python package: this one is not just an example, but the actual library you'd use if you're writing a server (or client)